db/migrate/20230821124053_create_tasks.rb

```diff
@@ -1,9 +1,9 @@
 class CreateTasks < ActiveRecord::Migration[7.0]
-  def change
+  def up
     create_table :tasks do |t|
       t.integer "user_id", :null => false
```

Comment:
this is a foreign key right ?

Reply:
if so please follow this format where you have to say if it is the foreign key or not
https://guides.rubyonrails.org/association_basics.html#the-has-many-association